modules/layers/src/path-layer/path-layer.js

```diff
@@ -190,6 +190,21 @@ export default class PathLayer extends Layer {
     return info;
   }
 
+  clearPickingColor(objectIndex) {
```

Comment:
Just a general nit: Overriding methods and calling `super` is in my opinion a quite subtle design mechanism and I find hard to follow when reading a complex code base (which picking certainly is).

 It is just hard to see in the calling code that the flow may involve code in a subclass as there is no hint in the base method, no trail to follow.

It may be too much work but perhaps separating out this into two functions could remove the `super` call. One that does the basic clearing and one that is overridable and calls the basic clearing function, rather than `super`